### PR TITLE
completed css-three-column-layout-exercise - Nick Sturz

### PR DESCRIPTION
**Pull request by nsturz**
Screen Shot 2022-06-30 at 8 18 28 PM

### CODE REVIEW

File: css-three-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <div class="row text-align justify-content">
+      <div class="circle">
+        <img src="images/merge.png" alt="">
+      </div>
+      <div class="circle">
+        <img src="images/cloud-computing.png" alt="">
+      </div>
+      <div class="circle">
+        <img src="images/discussion.png" alt="">
+      </div>
```
Comment (codingcodymiller) [inline css-three-column-layout/index.html:26]: As I had discussed with the class when introducing this challenge, this is a reasonable thought on how to divide the layout into rows, but this makes it seem like the images are unrelated to the text in the row below it. You should group the icon, subheader, and subtext all together in one `column`. Also, If you intend to use a `row`, the child elements should generally always have a `column` class to set up their size. There are exceptions, but it's almost always the case that a `column` is required.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Last review didn't register in the LMS, submitting one mroe

File: css-three-column-layout/index.html (hunk 1 above)
Comment (nsturz) [reply css-three-column-layout/index.html:26]: okay great, so just reiterating to make sure i understand - 

it looks like i need to create a row with 3 columns in it, and put a circle, subheader, and subtext in each column, right?

Comment (codingcodymiller) [reply css-three-column-layout/index.html]: Yup!

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  text-align: center;
+}
+
+.justify-content {
+  justify-content: space-around;
+}
+
+.column {
+  padding-left: 2%;
+}
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:47]: `column` classes should only have `flex-basis` applied to them. What size is this column supposed to be? You should give it an appropriate name and measurement.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-three-column-layout/styles.css [hunk 3]
```diff
@@ -42,7 +42,7 @@ h3 {
   justify-content: space-around;
 }
 
-.column {
+.one-third {
   padding-left: 2%;
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:46]: This should be named `column-one-third`, and it should have a declaration of `flex-basis`, and **_nothing else_**

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-three-column-layout/styles.css [hunk 4]
```diff
@@ -42,8 +42,8 @@ h3 {
   justify-content: space-around;
 }
 
-.one-third {
-  padding-left: 2%;
+.column-one-third {
+  flex-basis: auto%;
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:46]: `auto%` is not a value. If this is a one third width column, this should be `33%`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Pull request merged**